xml/System.Net.Http/HttpClient.xml

```diff
@@ -67,7 +67,29 @@ public class GoodController : ApiController
   
 ```  
   
-   
+The <xref:System.Net.Http.HttpClient> is a high-level API that wraps the lower-level functionality available on each platform where it runs.
+
+On each platform, <xref:System.Net.Http.HttpClient> tries to use the best available transport:
+
+| **Host/Runtime**  | **Backend**                                                                             |
+| ----------------- | --------------------------------------------------------------------------------------- |
+| Windows/.NET      | <xref:System.Net.HttpWebRequest>                                                        |
+| Windows/Mono      | <xref:System.Net.HttpWebRequest>                                                        |
+| Windows/UWP       | Windows native <xref:Syste.Net.Http.WinHttpHandler (HTTP 2.0 capable)                   |
+| Windows/.NET Core | Windows native <xref:Syste.Net.Http.WinHttpHandler (HTTP 2.0 capable)                   |
+| Android/Xamarin   | Build time selection;
+Can use either <xref:System.Net.HttpWebRequest> or be configured to use Android's native [`HttpURLConnection`](https://developer.xamarin.com/api/type/Java.Net.HttpURLConnection/) |
+| iOS, tvOS/Xamarin | Build time selection;
+Can use either <xref:System.Net.HttpWebRequest> or be configured to use Apple's [`NSUrlSession`](https://developer.xamarin.com/api/type/MonoTouch.Foundation.NSUrlSession/) (HTTP 2.0 capable) |
```

Comment:
this is not looking good at the built content. Perhaps remove the new line?

![image](https://user-images.githubusercontent.com/12971179/29087498-0a6d7e9c-7c2b-11e7-99cf-24bd960dd022.png)